src/scripts/clipperUI/panels/successPanel.tsx

```diff
@@ -9,12 +9,27 @@ import {ClipMode} from "../clipMode";
 import {ClipperStateProp} from "../clipperState";
 import {ComponentBase} from "../componentBase";
 import {Clipper} from "../frontEndGlobals";
+import {RatingsHelper, RatingsPromptStage} from "../ratingsHelper";
 import {Status} from "../status";
 
 import {SpriteAnimation} from "../components/spriteAnimation";
 
-class SuccessPanelClass extends ComponentBase<{ }, ClipperStateProp> {
-	onLaunchOneNoteButton() {
+import {DialogButton, DialogPanel} from "./dialogPanel";
+
+interface SuccessPanelState {
+	currentRatingsPromptStage?: RatingsPromptStage;
```

Comment:
I think the success panel itself does not interact with the ratings prompt portion. If so, we could keep those two completely decoupled (i.e., they don't know about each other). Then maincontroller can determine if the ratings prompt should be shown, and if so, show both success panel and ratings panel. That would also mean we can chain animations easier as it's handled in the parent container.